### ISSUE TITLE
Accessibility: Windows + NVDA does not read dates while navigating using arrow keys

### ISSUE THREAD

**Issue by MahogaWeerasinghe** [Hacker0x01/react-datepicker#3134]
Hi guys, 

Still, NVDA and Windows narrator doesn't read out the dates on navigating through date picker using arrow keys to me. So I like to know is this issue resolved or not ?

Previously mentioned this from another issue: **https://github.com/Hacker0x01/react-datepicker/issues/1624**

Thank you : )
